ci/kubetest/test_clickhouse_persistence_disabled.py

```diff
@@ -22,7 +22,11 @@ def setup(kube):
     wait_for_pods_to_be_ready(kube)
 
 
-def test_volume_claim(setup, kube):
+def test_helm_install(setup, kube):
```

Comment:
This looks a bit funny with the actual bit being tested in a fixture, could we instead run the commands within the test function?

I'm becoming more of the opinion that pytest fixture are cute, but not obvious